packages/manager/modules/vps/src/migration/components/plan/translations/Messages_fr_FR.json

```diff
@@ -0,0 +1,18 @@
+{
+  "vps_common_cancel": "Annuler",
+  "vps_common_confirm": "Confirmer",
+  "vps_migrate_schedule_title": "Migration du VPS: {{vpsName}}",
+  "vps_migrate_schedule_description": "Veuillez programmer la date et l’heure auxquelles vous souhaitez migrer votre ancien modèle de VPS vers la nouvelle gamme. Les options liées à votre VPS seront automatiquement migrées vers votre nouvelle machine.",
+  "vps_migrate_schedule_description_alert": "Attention, les anciennes sauvegardes enregistrées ne seront pas conservées",
+  "vps_migrate_schedule_to_model": "Nouveau modèle",
+  "vps_migrate_schedule_date": "Sélectionnez la date à laquelle vous souhaitez effectuer la migration",
+  "vps_migrate_schedule_time": "Sélectionnez l'heure à laquelle vous souhaitez effectuer la migration",
+  "vps_migration_plan_price": "{{price}}{{currencySymbol}}/mois",
```

Comment:
This kind of format won't fit for all languages. e.g.: UK format £20 (currency + price)